pkg/mimirtool/commands/alerts.go

```diff
@@ -109,9 +111,33 @@ func (a *AlertmanagerCommand) getConfig(_ *kingpin.ParseContext) error {
 		return err
 	}
 
-	p := printer.New(a.DisableColor)
+	if a.OutputDir == "" {
+		p := printer.New(a.DisableColor)
+		return p.PrintAlertmanagerConfig(cfg, templates)
+	}
+	return a.outputAlertManagerConfigTemplates(cfg, templates)
+}
 
-	return p.PrintAlertmanagerConfig(cfg, templates)
+func (a *AlertmanagerCommand) outputAlertManagerConfigTemplates(config string, templates map[string]string) error {
+	var baseDir string
+	baseDir, err := filepath.Abs(a.OutputDir)
+	if err != nil {
+		return err
+	}
+	log.Debug(fmt.Sprintf("writing the config file to %s", filepath.Join(baseDir, "config.yaml")))
```

Comment:
using Debugf is a bit birefer; same comment applies below
```suggestion
	log.Debugf("writing the config file to %s", filepath.Join(baseDir, "config.yaml"))
```